apps/prairielearn/src/lib/externalGraderCommon.js

```diff
@@ -203,7 +202,9 @@ module.exports.makeGradingResult = function (jobId, rawData) {
   } else if (_.isObject(rawData)) {
     data = JSON.stringify(rawData);
   }
-
+  /**
+   * @param {Object|string|Buffer} data - The grading results
+   */
```

Comment:
Looks like this got moved to a wrong line during a rebase perhaps?

Reply:
I had moved this while incorrectly thinking it was supposed to go above `data`. I have now moved this back to where it was and renamed to `rawData` to match the param.